exonum/src/node/requests.rs

```diff
@@ -71,26 +74,41 @@ impl NodeHandler {
 
     /// Handles `TransactionsRequest` message. For details see the message documentation.
     pub fn handle_request_txs(&mut self, msg: &Signed<TransactionsRequest>) {
-        use std::mem;
         trace!("HANDLE TRANSACTIONS REQUEST");
+        self.send_transactions_by_hash(&msg.author(), msg.txs());
+    }
+
+    /// Handles `PoolTransactionsRequest` message. For details see the message documentation.
+    pub fn handle_pool_request_txs(&mut self, msg: &Signed<PoolTransactionsRequest>) {
```

Comment:
I'd name it like this:
```suggestion
    pub fn handle_request_pool_txs(&mut self, msg: &Signed<PoolTransactionsRequest>) {
```